### ISSUE TITLE
Image alt text

### ISSUE THREAD

**Issue by justintadlock** [Codeinwp/zillah#56]
In `inc/widgets/about-me.php`, the image output is missing the `alt` attribute.  Even if not using alt text, you should add an empty `alt` attribute:

```
<img class="about-photo" src="' . $image_url . '"/>
```

**Comment by rodica-andronache** [comment]
@claudiuvertistudio I think you need to check here if the $title variable exists before using in as alt attribute. And if it's empty the alt should be empty

**Comment by rodica-andronache** [comment]
@claudiuvertistudio I think you need to change it to this:
